README.md

```diff
@@ -0,0 +1,14 @@
+# Juju Machine Exporter
```

Comment:
this will need to be updated for the new snap name

Reply:
Thanks for pointing it out. I've fixed the typos.